README.md

```diff
@@ -25,7 +25,7 @@ pyPESTO features include:
 * Parameter estimation pipeline for systems biology problems specified in
   [SBML](http://sbml.org/) and [PEtab](https://github.com/PEtab-dev/PEtab)
   ([example](https://github.com/ICB-DCM/pyPESTO/blob/master/doc/example/petab_import.ipynb))
-* Parameter estimation with qualitative data as described in
+* Parameter estimation with ordinal data as described in
```

Comment:
I think consensus was to use "qualitative".

Reply:
I don't remember anymore. 
In any case, I would still vote for ordinal. I think it's a more exact and descriptive name. Additionally, that's what it's called on Wikipedia :D https://en.wikipedia.org/wiki/Ordinal_data

Reply:
Ordinal and qualitative are two different things - ordinal requires a relative ordering (x, y) -> x<y. What Leonard assumed was at least such an ordering, so I'd say ordinal is more accurate?

Reply:
Daniel asked me to consider Jan as well, he confirmed ordinal would be more accurate.